docs/modules/index.md

```diff
@@ -19,12 +19,3 @@ We have C# support available in experimental status. C# can be a good choice for
 
 - [C# Module Reference](/docs/modules/c-sharp)
 - [C# Module Quickstart Guide](/docs/modules/c-sharp/quickstart)
-
```

Comment:
This makes redundant https://github.com/clockworklabs/spacetime-docs/pull/197. Closes #197

Reply:
Sorry to step on your toes!